requirements.txt

```diff
@@ -4,6 +4,7 @@ bleach==1.4.3  # Apache 2.0
 boto3==1.4.4  # Apache 2.0
 celery==3.1.25  # BSD 3 Clause
 colorlog==2.7.0  # MIT
+coreapi==2.3.1  # BSD
```

Comment:
Hm, whoops, I installed coreapi to check out DRF's [auto-generated docs](http://www.django-rest-framework.org/topics/documenting-your-api/) (which apparently chokes on some of our more unconventional API code, alas), and could have sworn the markdown rendering stopped working when I uninstalled it. I guess I was confused.